csharp/ql/src/Security Features/CWE-022/ZipSlip.ql

```diff
@@ -0,0 +1,91 @@
+/**
+ * @name Potential ZipSlip vulnerability 
+ * @description When extracting files from an archive, don't add archive item's path to the target file system path. Archive path can be relative and can lead to 
+ * file system access outside of the expected file system target path, leading to malicious config changes and remote code execution via lay-and-wait technique 
+ * @kind problem
+ * @id cs/zipslip
+ * @problem.severity error
+ * @precision high
```

Comment:
Could you remove this line containing the  `@precision` tag. We are introducing a new workflow whereby we will add the `@precision` tag later when the query is ready to be deployed by default on LGTM.